src/pages/wow/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
This `wow` directory appears to be unused

Reply:
yeah it was just to test that we could add more pages in the future, will delete in next commit